src/Mod/TechDraw/Gui/QGISVGTemplate.cpp

```diff
@@ -132,12 +197,30 @@ void QGISVGTemplate::draw()
 
 void QGISVGTemplate::updateView(bool update)
 {
-    Q_UNUSED(update);
+    if (update) {
+        clearClickHandles();
+        createClickHandles();
+    }
     draw();
 }
 
+void QGISVGTemplate::clearClickHandles()
+{
+    prepareGeometryChange();
+    constexpr int TemplateTextFieldType{QGraphicsItem::UserType + 160};
```

Comment:
What is this this magic number `160`?

Reply:
Code from @WandererFan

Reply:
See QGIUserType.h

Every QGraphicsItem must have a unique type number (int).  Type numbers below QGraphicsItem::UserType belong to Qt for base QGraphicsItems.  Derived QGraphicsItems use type numbers > QGraphicsItem::UserType.